js/constants/constants.js

```diff
@@ -148,7 +148,8 @@ module.exports = {
 
   UI: {
     defaultScrubberBarHeight: 4,
-    DEFAULT_SCRUBBERBAR_LEFT_RIGHT_PADDING: 15
+    DEFAULT_SCRUBBERBAR_LEFT_RIGHT_PADDING: 15,
+    BUFFERING_SPINNER_DELAY: 1250
```

Comment:
Can you add a comment specifying that this is in ms?

Reply:
Sure, although I think this is actually going to end up in the skin.json